wondrous-app/components/Common/NewInviteLinkModal/styles.tsx

```diff
@@ -0,0 +1,374 @@
+import {
+  Modal,
+  Box,
+  Typography,
+  TextField,
+  Button,
+  Divider,
+  Select,
+  Menu,
+  MenuItem,
+  FormControl,
+  FormControlLabel,
+  InputLabel,
+} from '@mui/material';
+import styled from 'styled-components';
+import { ModalCloseButton } from '../ModalCloseButton';
+import { Button as ButtonComponent } from '../button';
+import { AndroidSwitch } from '../../CreateEntity/createEntityModal';
+import palette from 'theme/palette';
+import { capitalize } from 'lodash';
+
+export const StyledModal = styled(Modal)``;
+
+export const StyledBox = styled(Box)`
+  width: 682px;
+  border-radius: 6px;
+  background: linear-gradient(180deg, #1e1e1e 0%, #141414 100%);
+  margin: 140px auto 0;
+  padding: 26px;
+  position: relative;
+
+  &::before {
+    content: '';
+    position: absolute;
+    inset: 0;
+    background: linear-gradient(169.47deg, rgba(75, 75, 75, 0.5) 7.84%, rgba(35, 35, 35, 0.5) 108.71%);
+    -webkit-mask: linear-gradient(#fff 0 0) content-box, linear-gradient(#fff 0 0);
```

Comment:
do you need these `-webkit-mask` and `-webkit-mask-composite` ?
These features are non-standard.

Reply:
please remove `-webkit-mask` and `-webkit-mask-composite`